tests/page/page-network-request.spec.ts

```diff
@@ -289,8 +288,8 @@ it('should report all headers', async ({ page, server, browserName, platform })
       ]
     }))
   ]);
-  const headers = await request.allHeaders();
-  expect(headers.headers()).toEqual(expectedHeaders);
+  const headers = await request.headersArray();
+  expect(headers.sort()).toEqual(expectedHeaders.sort());
```

Comment:
fetch API actually sends headers to server like below:

```
headers-a: value-a, value-a-1, value-a-2
headers-b: value-b
```

and headersArray() returns

```
[
  ['headers-a', 'value-a, value-a-1, value-a-2'],
  ['headers-b', 'value-b'],
]
```

Is it expected...?

I thought this test want to check the behavior with headers like

```
header-a: value-a
header-b: value-b
header-a: value-a-1
header-a: value-a-2
```

however it seems JS fetch API doesn't allow to send such request...